jobsdb/jobsdb.go

```diff
@@ -287,6 +287,7 @@ type JobsDB interface {
 	Status() interface{}
 	Ping() error
 	DeleteExecuting()
```

Comment:
what would the implications be if we failed executing in processor too?

Reply:
The processor only looks for unprocessed jobs, and marks them succeeded in any case.
Failed jobs wouldn't be picked at all by it when the server starts up.
There was not much motivation in this particular case to preserve the statuses because each job is picked only once(unless of course the server crashes) and marked `succeeded` anyway.